packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -418,12 +427,14 @@ class InspectorController extends DisposableController
     }
 
     if (flutterAppFrameReady) {
-      // TODO: measure and send DevTools pageReady analytics:
-      // https://github.com/flutter/devtools/issues/3879
-      await serviceManager.sendDwdsEvent(
-        screen: InspectorScreen.id,
-        action: analytics_constants.pageReady,
-      );
+      if (serviceManager.connectedApp!.isDartWebAppNow!) {
```

Comment:
@annagrin @elliette is this the proper check that will ensure dwds is available and that the `ext.dwds.sendEvent` service has been registered?

This was failing for a flutter mobile app and was causing https://github.com/flutter/devtools/issues/3930. I removed the await so that sending this event does not block the inspector loading.